src/aics-image-viewer/components/App/savedState.ts

```diff
@@ -1,43 +0,0 @@
-import { ChannelState } from "../../shared/utils/viewerChannelSettings";
-import { ShowControls, GlobalViewerSettings } from "./types";
-
-/** Global viewer settings that are not currently controlled by any UI */
-// TODO double-check that this is exhaustive
-interface View3dInternalGlobalSettings {
-  pixelSamplingRate: number;
-  stepSizePrimaryRayVoxels: number;
-  stepSizeSecondaryRayVoxels: number;
-  flipVolume: [number, number, number];
-}
```

Comment:
Might have missed this during my review, but what happened to these settings? Did they get moved or just deleted?

Reply:
This is a pretty old pass I took at viewer state types geared towards some sort of serializable format. I think I included it, maybe accidentally, in a commit a long time ago but never actually used it.